tox.ini

```
@@ -18,3 +18,7 @@ commands =
 
 whitelist_externals =
     make
+
+[testenv:pyw3]
```

Perhaps
```suggestion
[testenv:py3_mingw]
```
would be clearer